spec/controllers/api/groups_controller_spec.rb

```diff
@@ -587,5 +598,70 @@
         expect(annotation_contents).to contain_exactly('Content 1', 'Content 2')
       end
     end
+    context 'PUT remove_tag' do
+      let(:response_type) { 'application/xml' }
+      let(:tag) { create :tag, assessment: assignment }
+      let(:grouping) { create :grouping, group: group, tags: [tag], assignment: assignment }
+
+      before do
+        request.env['HTTP_ACCEPT'] = response_type
+      end
+
+      it 'should let the user remove a tag' do
+        put :remove_tag, params: { id: grouping.group.id, assignment_id: assignment.id,
+                                   course_id: course.id, tag_id: tag.id }
+        expect(response.status).to be(200)
```

Comment:
use `have_http_status` when checking HTTP status codes (throughout the tests you added)